Ginger/GingerCoreNET/ActionsLib/ActUIElement.cs

```diff
@@ -116,6 +116,8 @@ public new static partial class Fields
             public static string WaitforIdle = "WaitforIdle";
             public static string IsWidgetsElement = "IsWidgetsElement";
             public static string IsMouseEvent = "IsMouseEvent";
+            public static string POMElementLocator = "POMElementLocator";
```

Comment:
we already have fields for locate by and locate value what are these fields added for?